tests/ui/const-generics/const-param-type-depends-on-const-param.full.stderr

```diff
@@ -3,12 +3,16 @@ error[E0770]: the type of const parameters must not depend on other generic para
    |
 LL | pub struct Dependent<const N: usize, const X: [u8; N]>([(); N]);
    |                                                    ^ the type must not depend on the parameter `N`
+   |
+   = note: const parameters may not be used in the type of const parameters
```

Comment:
```suggestion
   = note: const parameters may not be used in the type of other const parameters
```
?

Reply:
I think technically no because const parameters cannot be used in the type of themselves either- `const N: [(); N]` would error.